src/variants.php

```diff
@@ -823,6 +824,31 @@ function(sData) {
                 $_POST['type'] = NULL;
             }
 
+            // Add the source of the variant if a source can be found.
+            if (empty($_POST['source'])) {
+                // If nothing is given, the source is set to null, meaning that the source is unknown.
+                unset($aFieldsGenome[array_search('source', $aFieldsGenome)]);
```

Comment:
Dit is prima en kun je zeker zo laten staan, omdat we weten dat `source` zich in `$aFieldsGenome` bevindt. Denk er alleen wel aan dat als `source` zich mogelijk niet in `$aFieldsGenome` bevindt, dat we hier een foutmelding zouden krijgen.